<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_add_task</class>
 <widget class="QDialog" name="dlg_add_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/animal-penguin.png</normaloff>icons/animal-penguin.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="name_add_task">
       <property name="placeholderText">
        <string>Наименование...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="date_add_task"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="but_add_task">
       <property name="text">
        <string>Добавить задачу</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
